### ISSUE TITLE
[RFE] add disk selection

### ISSUE THREAD

**Issue by natman** [zipurman/oVIRT_Simple_Backup#52]
#### Expected behaviour
should be good to select disk to backup when saving and restoring
#### Actual behaviour
we have to backup/restore the full disks stack
#### Steps to reproduct the behaviour
need a checkbox to do so
thank you

**Comment by zipurman** [comment]
The backup/restore is currently setup to backup an entire VM and restore an entire VM. It was never planned to backup selected disks only and then restore only selected disks back to a working VM. The restore creates an entirely new VM based on the backup.

**Comment by natman** [comment]
I agree, but I had an issue with a 3 disks (1 system and 2 data disks) vm to restore, when failing the logs said that I couldn't restore the 3 disks at the same time. So in that case, it was impossible to restore a vm with at least the only system disk. That's why I had to use the uploading API so as to at least upload the system disk.

**Comment by zipurman** [comment]
So you did a backup using simpleBackup of a 3 disk VM and then when you went to restore the backup it failed? If so, do you have the logs for the restore?
